modules/sysrepo-factory-default@2024-05-02.yang

```diff
@@ -18,19 +18,31 @@ module sysrepo-factory-default {
     description
         "Sysrepo YANG augment of the factory-reset RPC.";
 
+    revision "2024-05-02" {
+        description
+            "Add datastore leaf-list parameter.";
+    }
+
     revision "2023-02-23" {
         description
             "Initial revision.";
     }
 
     augment /fd:factory-reset/fd:input {
         description
-            "Adds leaf-list of modules that are reset to the default factory state. If there are none, sysrepo
-             automatically populates the leaf-list with all the modules being reset.";
+            "Adds leaf-list of modules and datastores that are reset to the default factory state. If there are none, sysrepo
+             automatically populates the leaf-lists with all the modules/datastores being reset.";
```

Comment:
Would be good to specify what "all" datastores mean (`running`, `startup`, and `candidate`).